source/playgrounds.txt

```diff
@@ -177,6 +181,26 @@ of your playground.
 immediately above your selected section. Click this link to test and
 troubleshoot specific lines or sections of your playground.
 
+Code Formatting and Linting Tools
+---------------------------------
+
+MongoDB Playgrounds are proper JavaScript files and can work with popular
+|vscode-short| formatting and linting extensions like
+`Prettier <https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode>`_ 
+and `ESLint <https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint>`_.
+
+If you are using a code formatting extension, MongoDB Playgrounds 
```

Comment:
[suggestion]

If you use a code formatting extension..."

Reply:
Updated.